SPECS/jasper/jasper.spec

```diff
@@ -62,6 +63,7 @@ Requires: %{name}-libs%{?_isa} = %{version}-%{release}
 
 %prep
 %setup -q -n %{name}-version-%{version}
+%patch 0 -p1
```

Comment:
While we're here:
```suggestion
%autosetup -q -n %{name}-version-%{version} -p1
```

Reply:
It's applying other patches based on architecture, so I added an explicit patch line